get_github_coderabbit_review/coderabbit_fetcher/cli/parser.py

```
@@ -0,0 +1,188 @@
+            
+        Raises:
+            InvalidPRUrlError: If PR URL is invalid
+            PersonaFileError: If persona file is invalid
+        """
+        # Validate PR URL
+        self.parse_pr_url(pr_url)
+        
+        # Validate persona file if provided
+        if persona_file and not persona_file.exists():
+            from ..exceptions import PersonaFileError
+            raise PersonaFileError(f"Persona file not found: {persona_file}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)